History.md

```diff
@@ -1,3 +1,5 @@
+* HeathS: Fix variadic macros for custom action messages
```

Comment:
Please don't edit History.md directly any longer. Use the history\ folder instead.